webapp/components/admin_console/push_settings.jsx

```diff
@@ -207,6 +207,7 @@ export default class PushSettings extends AdminSettings {
                 <DropdownSetting
                     id='pushNotificationContents'
                     values={[
+                        {value: 'generic_no_channel', text: Utils.localizeMessage('admin.email.genericNoChannelPushNotification', '"Send generic description with only sender name')},
                         {value: 'generic', text: Utils.localizeMessage('admin.email.genericPushNotification', 'Send generic description with user and channel names')},
```

Comment:
@coreyhulen The new help text doesn't match the option in the dropdown. It has "Send generic description with sender and channel names", but the dropdown option uses "user and channel names"